docs/source/usage/other.rst

```diff
@@ -17,6 +17,9 @@ If installed using pip just add
 
 (replace ``python`` with ``python3`` if appropriate) to the :file:`vimrc`.
 
+.. note::
+    Status line will not appear by default when there is only a single window displayed. `:h laststatus` for more information.
```

Comment:
Tag is named `'laststatus'` (with single quotes), there is no `laststatus` tag: it works only because Vim also does substring matching. Code should be enclosed in *two* backticks, not one. Line too long, wrap at 80’th column.

Also many other notes are aligned with `n` in `note`, though not all AFAIK, can’t say what is more common. I align with `n` when I remember to align with `n`.

// And information is already in troubleshooting, third Vim issue.